GreenGarbageApp/app/src/main/java/com/example/greengarbageapp/activities/MainActivity.kt

```diff
@@ -1,11 +1,15 @@
 package com.example.greengarbageapp.activities
 
+import android.app.SearchManager
+import android.content.DialogInterface
+import android.content.Intent
 import android.media.AudioManager
 import android.media.MediaPlayer
 import android.os.Bundle
 import android.view.View
-import android.widget.Button
+import android.widget.ImageButton
 import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
```

Comment:
Could this be better implemented with a DialogBox instead?

Reply:
The library uses the the Alert dialog box that's why I used the same.